src/app/themes.tsx

```diff
@@ -1,60 +1,153 @@
+            largeForeground: BLACK,
+        },
+        darkYellow: {
+            background: "#ffbb00",
+            foreground: BLACK,
+            largeForeground: BLACK,
+        },
+        lightGreen: {
+            background: "#b4f5b3",
+            foreground: BLACK,
+            largeForeground: BLACK,
+        },
+        darkGreen: {
+            background: "#5fa881",
+            foreground: BLACK,
+            largeForeground: BLACK,
+        },
+        lightBlue: {
+            background: "#acd5df",
+            foreground: BLACK,
+            largeForeground: BLACK,
+        },
+        darkBlue: {
+            background: "#7bbafc",
+            foreground: BLACK,
+            largeForeground: BLACK,
+        },
+        lightPurple: {
+            background: "#ceaae9",
+            foreground: BLACK,
+            largeForeground: BLACK,
+        },
+        darkPurple: {
+            background: "#b18ff9",
+            foreground: BLACK,
+            largeForeground: BLACK,
+        },
+        lightGrey: {
+            background: "#d7d7d7",
+            foreground: BLACK,
+            largeForeground: BLACK,
+        },
+        darkGrey: {
+            background: "#808080",
+            foreground: BLACK,
+            largeForeground: WHITE,
+        },
+        lightBrown: {
+            background: "#bfaba2",
+            foreground: BLACK,
+            largeForeground: BLACK,
+        },
+        darkBrown: {
+            background: "#a68f68",
+            foreground: BLACK,
+            largeForeground: BLACK,
+        },
+    },
 };
 
-const darkTheme: CustomTheme = {
-    foregroundColor: "#ffffff",
-    backgroundColor: "#1a1a1a",
-    primaryForegroundColor: "#ffffff",
-    primaryBackgroundColor: "#066940",
-    secondaryForegroundColor: "#000000",
-    secondaryBackgroundColor: "#63a036",
-    accentBackgroundColor: "#1b629c",
-    accentForegroundColor: "#ffffff",
-    surfaceForegroundColor: "#ffffff",
-    surfaceBackgroundColor: "#161414",
-    errorColor: "#ff624e",
+export const darkTheme: DefaultTheme = {
+    ...lightTheme,
+    light: false,
+    main: {
+        background: ["#262626", "#282828", "#2d2d2d", "#363636"],
+        foreground: ["#d9d9d9", "#d9d9d9", "#d9d9d9", "#d9d9d9"],
+        largeForeground: ["#d9d9d9", "#d9d9d9", "#d9d9d9", "#d9d9d9"],
+        lighterForeground: ["#969696", "#969696", "#969696", "#bbbbbb"],
+        border: "#5c5c5c",
+    },
+    primary: {
+        background: [...lightTheme.primary.background].reverse(),
+        foreground: [...lightTheme.primary.foreground].reverse(),
+        largeForeground: [...lightTheme.primary.largeForeground].reverse(),
+    },
+    accent: {
+        background: "#b8cbe9",
+        foreground: "#2d2d2d",
+        largeForeground: "#2d2d2d",
+    },
+    shadow: "#282828",
+    error: "#ff7361",
 };
 
 interface Props {
-    children: React.ReactElement;
-    theme?: CustomTheme;
+    children: React.ReactNode;
```

Comment:
ReactElement is better than ReactNode where possible - I'd rather specify that this is a ReactElement and catch when we accidently put in `null` or something else!